packages/connect-explorer/src/components/Settings.tsx

```diff
@@ -0,0 +1,49 @@
+import styled from 'styled-components';
+
+import { Button } from '@trezor/components';
+
+import * as trezorConnectActions from '../actions/trezorConnectActions';
+import { useSelector, useActions } from '../hooks';
+
+import { Row } from './fields/Row';
+import { getField } from './Method';
+import { Field, FieldWithBundle } from '../types';
+
+const SettingsContent = styled.section`
+    flex: 1;
+    padding: 10px 20px;
+    display: flex;
+    flex-direction: column;
+`;
+
+export const Settings = () => {
+    const connectOptions = useSelector(state => ({
+        trustedHost: state.connect?.options?.trustedHost || false,
+        transports: state.connect?.options?.transports || 'bridge',
```

Comment:
'bridge' does not really match what is allowed by types. maybe types are not strict enough here

Reply:
Yes, let's be simple and use only `trustedHost` now, then we can extend it 336857ebe8d864e45497fedd054b8bd65c0641b3